server/groundstation.py

```diff
@@ -0,0 +1,97 @@
+import json
+import time
+from threading import Thread
+
+from interop_handler import InteropHandler
+from uav_handler import UAVHandler
+from dummy_uav_handler import DummyUAVHandler
+
+
+class GroundStation:
+
+    def __init__(self, socket):
+        with open("config.json", "r") as file:
+            self.config = json.load(file)
+
+        self.interop = InteropHandler(self, config=self.config)
+        self.interop_telem_thread = self.update_thread = None
+
+        dummy_uav = self.config["uav"]["dummy"]
+
+        if dummy_uav:
+            self.uav = DummyUAVHandler(self, self.config, socket)
```

Comment:
```suggestion
        if self.config["uav"]["dummy"]:
            self.uav = DummyUAVHandler(self, self.config, socket)
```
Just a tad bit more concise.